<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageBrowserWidget</class>
 <widget class="QWidget" name="StorageBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Storage Container:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="StorageContainer">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="accessibleName">
        <string>Storage container</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AddContainerButton">
       <property name="toolTip">
        <string>Create new storage container</string>
       </property>
       <property name="accessibleName">
        <string>Create Storage Container</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="new">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DeleteContainerButton">
       <property name="toolTip">
        <string>Delete active storage container</string>
       </property>
       <property name="accessibleName">
        <string>Delete Storage Container</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="RefreshButton">
       <property name="toolTip">
        <string>Refresh display of storage container contents</string>
       </property>
       <property name="accessibleName">
        <string>Refresh Storage Container</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="AddFolderButton">
       <property name="toolTip">
        <string>Create new folder in currently selected folder.</string>
       </property>
       <property name="accessibleName">
        <string>Add Folder...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="newfolder">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DeleteItemButton">
       <property name="toolTip">
        <string>Delete currently selected file or folder.</string>
       </property>
       <property name="accessibleName">
        <string>Delete Selected Item</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="session_error">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UploadFileButton">
       <property name="toolTip">
        <string>Upload one or multiple files into the currently selected folder.</string>
       </property>
       <property name="text">
        <string>Upload File...</string>
       </property>
       <property name="icon">
        <iconset theme="upload">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UploadFolderButton">
       <property name="toolTip">
        <string>Upload a folder with all its contents into the currently selected folder.</string>
       </property>
       <property name="text">
        <string>Upload Folder...</string>
       </property>
       <property name="icon">
        <iconset theme="directory">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="FileTree">
     <property name="accessibleName">
      <string>File structure in selected container</string>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ContentEditGroup" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>StorageContainer</tabstop>
  <tabstop>AddContainerButton</tabstop>
  <tabstop>DeleteContainerButton</tabstop>
  <tabstop>RefreshButton</tabstop>
  <tabstop>AddFolderButton</tabstop>
  <tabstop>DeleteItemButton</tabstop>
  <tabstop>UploadFileButton</tabstop>
  <tabstop>UploadFolderButton</tabstop>
  <tabstop>FileTree</tabstop>
 </tabstops>
 <resources>
  <include location="../../Resources/ArrtApplication.qrc"/>
 </resources>
 <connections/>
</ui>
